pallets/dkg/src/functions.rs

```diff
@@ -295,6 +316,32 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Verifies the DKG signature result for BLS signatures.
+	///
+	/// This function uses the BLS signature algorithm to verify the provided signature
+	/// based on the message data, signature, and signing key in the DKG signature result.
+	///
+	/// # Arguments
+	///
+	/// * `data` - The DKG signature result containing the message data, BLS signature, and signing
+	///   key.
+	fn verify_bls_signature(data: DKGTSSSignatureResult) -> DispatchResult {
+		let public_key = blst::min_pk::PublicKey::deserialize(&data.signing_key)
+			.map_err(|_err| Error::<T>::InvalidParticipantPublicKey)?;
```

Comment:
Not a really helpful error used here, maybe add a new error instead of reusing this one.